src/linear_algebra/finsupp.lean

```diff
@@ -537,6 +537,11 @@ begin
     refine sum_mem _, simp [this] }
 end
 
+/-- Special case of `span_image_eq_map_total`. -/
+lemma span_range_eq_map_total :
+  span R (set.range v) = submodule.map (finsupp.total α M R v) (supported R R set.univ) :=
```

Comment:
Why not go further to
```suggestion
  span R (set.range v) = (finsupp.total α M R v).range :=
```

Reply:
Oh, we already have that lemma! It's `range_total`, in the other direction.

I think the thing to do here is to add an alias of `range_total` called `span_range` which is just the reversed direction.

Reply:
Oh you're right @eric-wieser , `rw [←finsupp.range_total, linear_map.mem_range],` works quite well for me. Is there a reason that `range_total` is not called `range_total_eq_span_range`? 

(statement is `range_total : (finsupp.total α M R v).range = span R (range v)`)